### PR TITLE
[RFR] Example of admin implementation

### PR DESCRIPTION
**Pull request by ndufreche**
This is example of admin bundle implementation.

Is actually not tested nor finished and it's based on a class that not exist actually in backbee/BackBee. It's just a basic admin to permit to talk over on this implementation.

### CODE REVIEW

File: Controller/AdminController.php [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        $itemRepo = $this->getRepository('BackBee\Bundle\DemoBundle\Entity\Item');
+        $id = $this->getRequest()->request->get('id');
+
+        if (is_null($id)) {
+            $item = new Item();
+        } else {
+            $item = $itemRepo->findEntities($id);
+        }
+
+        $item->setLabel($this->getRequest()->request->get('label')
```
Comment (mickaelandrieu) [inline Controller/AdminController.php]: this cant work :p

File: Controller/AdminController.php [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+{
+    /**
+     * Item index
+     *
+     * @return [string] [Index template]
+     */
+    public function indexAction()
+    {
+        $items = $this->getRepository('BackBee\Bundle\DemoBundle\Entity\Item')
+                      ->findAllEntities();
```
Comment (fkroockmann) [inline Controller/AdminController.php]: findAll instead of findAllEntities

File: Controller/AdminController.php [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+    /**
+     * Edit  item form
+     *
+     * @param  [Int] $id [item id]
+     * @return [string]  [form]
+     */
+    public function editAction($id)
+    {
+        $item = $this->getRepository('BackBee\Bundle\DemoBundle\Entity\Item')
```
Comment (fkroockmann) [inline Controller/AdminController.php]: Just find

File: Entity/Item.php [hunk 4]
```diff
@@ -0,0 +1,98 @@
+<?php
+namespace BackBee\Bundle\LayoutBuilderBundle\Entity;
```
Comment (mickaelandrieu) [inline Entity/Item.php]: Can we use real entities in bundle ? which create real tables with Doctrine for instance ?

File: Controller/AdminController.php (hunk 2 above)
Comment (ndufreche) [inline Controller/AdminController.php]: Nop this is a Registry Entity :)

File: Controller/AdminController.php [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+     */
+    public function saveAction()
+    {
+        $itemRepo = $this->getRepository('BackBee\Bundle\DemoBundle\Entity\Item');
+        $id = $this->getRequest()->request->get('id');
+
+        if (is_null($id)) {
+            $item = new Item();
+        } else {
+            $item = $itemRepo->findEntities($id);
```
Comment (fkroockmann) [inline Controller/AdminController.php]: findEntity ?

File: Entity/Item.php (hunk 4 above)
Comment (fkroockmann) [inline Entity/Item.php]: I agree with that

Comment (ndufreche) [inline Entity/Item.php]: Yes we can use doctrine. This just an example of EntityRegistry implementation => this system is here just to record some Entity where the number of row don't justify a table creation.

File: Config/config.yml [hunk 6]
```diff
@@ -5,3 +5,6 @@ bundle:
     version: 1
     enable: true
     config_per_site: false
+
+    controller:
```
Comment (eric-chau) [inline Config/config.yml]: We need a more explicit key name than ``controller`` to show that this is used by toolbar front js.

Comment (ndufreche) [inline Config/config.yml]: @eric-chau what did you propose ?

Comment (eric-chau) [inline Config/config.yml]: Hm I don't know ^^'

**Comment by crouillon** [comment]
:+1:

**Comment by mickaelandrieu** [comment]
Thank you for review @crouillon, good job @ndufreche !

**Pull request merged**